Content.Server/Cargo/Systems/CargoSystem.Bounty.cs

```diff
@@ -434,15 +437,15 @@ public bool TryAddBounty(EntityUid uid, CargoBountyPrototype bounty, StationCarg
     }
 
     [PublicAPI]
-    public bool TryRemoveBounty(EntityUid uid, string dataId, StationCargoBountyDatabaseComponent? component = null)
+    public bool TryRemoveBounty(EntityUid uid, string dataId, StationCargoBountyDatabaseComponent? component = null, EntityUid? actor = null)
```

Comment:
Since you're already changing the method signature, it would be nice if you could convert it to the [new way](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#public-api-method-signature) we manage passing components into functions.

```suggestion
    public bool TryRemoveBounty(Entity<StationCargoBountyDatabaseComponent?> ent, string dataId, EntityUid? actor = null)
```

Reply:
I'll try and figure this one out.

Reply:
Feel free to ask for guidance on the discord as there’s not much documentation for it and there’s a lot of outdated code using the old method.